app/views/statuses/_task_fields.html.erb

```diff
@@ -0,0 +1,19 @@
+<div class="nested-fields mb-5">
+  <div class= "form-row">
+    <div class="form-group col-md-4 mb-5">
+      <%= f.label "Completed Tasks", class: "input-group" %>
```

Comment:
Use single quotes for string when there is no special characters and string interpolation, Fix wherever required